mik

```diff
@@ -20,8 +20,8 @@ if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
 // in the set_error_handler function halting the program.
 // Provide a default timezone if date.timezone is null in the the PHP INI.
 if (null == ini_get('date.timezone')) {
-    date_default_timezone_set('America/Vancouver');
-};
+    date_default_timezone_set('America/Chicago');
+}
```

Comment:
This shouldn't be hard coded for you guys or for SFU. I've opened [an issue](https://github.com/MarcusBarnes/mik/issues/215) to address this. Can you remove this change from your PR?